bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/command/GeyserFabricCommandExecutor.java

```diff
@@ -51,11 +51,13 @@ public boolean testPermission(CommandSourceStack source) {
         return Permissions.check(source, command.permission(), command.isSuggestedOpOnly() ? 2 : 0);
     }
 
+    @SuppressWarnings("rawtypes")
     @Override
     public int run(CommandContext context) {
         return runWithArgs(context, "");
     }
 
+    @SuppressWarnings("rawtypes")
     public int runWithArgs(CommandContext context, String args) {
         CommandSourceStack source = (CommandSourceStack) context.getSource();
         FabricCommandSender sender = new FabricCommandSender(source);
```

Comment:
```suggestion
    @Override
    public int run(CommandContext<CommandSourceStack> context) {
        return runWithArgs(context, "");
    }

    public int runWithArgs(CommandContext<CommandSourceStack> context, String args) {
        CommandSourceStack source = context.getSource();
```
looks like the raw typing can just be fixed